app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -56,13 +59,28 @@ public function panel(Panel $panel): Panel
             ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
             ->discoverClusters(in: app_path('Filament/Clusters'), for: 'App\\Filament\\Clusters')
             ->pages([
-                // Pages\Dashboard::class,
+                //Pages\Dashboard::class,
+                Settings::class,
```

Comment:
Should be in Pages\

Reply:
For some reason it does not work in Pages. If i put it there I get an error that the route is not known.